jupyterlab-mlops/Dockerfile

```diff
@@ -41,13 +72,12 @@ RUN mamba install --quiet --yes kfp jupyter-server-proxy -c conda-forge && \
 # configure python 3.8 env with kedro
 RUN mamba create --quiet --yes -p "${CONDA_DIR}/envs/python38" python=3.8 ipython ipykernel kedro=$KEDRO_VERSION && \
     mamba clean --all -f -y && \
-    "${CONDA_DIR}/envs/python38/bin/python" -m ipykernel install --user --name=python38 && \
+    "${CONDA_DIR}/envs/python38/bin/python" -m ipykernel install --user --name=kedro-python38 && \
     fix-permissions "/home/${NB_USER}" && \
     fix-permissions "${CONDA_DIR}"
-    
+# does above kernel installation make any sense in case of mounting home dir ?
```

Comment:
Nope, in case of home dir remount it doesn't make any sense. It works only in ephemeral KF notebooks

Reply:
got it, comment reformulated.